src/tests/DomainHost.t.sol

```diff
@@ -470,39 +511,34 @@ contract DomainHostTest is DssTest {
         host.lift(100 ether);
         host.cage();
 
-        assertEq(host.cure(), 0);
-        assertTrue(!host.cureReported());
+        uint256 initialVatDai = vat.dai(vow);
+        uint256 initialHostGrain = host.grain();
+        assertTrue(!host.debtReported());
 
         vm.expectEmit(true, true, true, true);
         emit Tell(100 * RAD);
         host.tell(0, 100 * RAD);
 
-        assertEq(host.cure(), 100 * RAD);
-        assertTrue(host.cureReported());
+        assertEq(vat.dai(vow), initialVatDai + 100 * RAD - initialHostGrain * RAY);
+        assertEq(host.grain(), 100 ether);
+        assertTrue(host.debtReported());
+        assertEq(host.debt(), 100 * RAD);
     }
 
     function testTellNotCaged() public {
         host.lift(100 ether);
 
         vm.expectRevert("DomainHost/live");
-        host.tell(0, 100 * RAD);
-    }
-
-    function testTellCureBadValue() public {
-        host.lift(50 ether);
-        host.cage();
-
-        vm.expectRevert("DomainHost/cure-bad-value");
-        host.tell(0, 100 * RAD);
+        host.tell(0, 100 ether);
```

Comment:
It doesn't really matter for this test, but the value should be a `RAD`.

Reply:
Yes you are right, there was an intermediate state where `debt` was defined as `wad`, probably forgot to update it, will fix it.

Reply:
Done.